src/pybind/mgr/dashboard/frontend/src/app/shared/components/wizard/wizard.component.scss

```diff
@@ -8,6 +8,10 @@
   padding-left: 0;
 }
 
+.no-wrap {
+  white-space: nowrap;
+}
+
```

Comment:
Not sure about this, I think it could end up causing more trouble for other wizards

Reply:
This was required to be changed for the multisite wizard, I have checked the Expand Cluster Wizard and its not affected by this.

Reply:
why not use the bootstrap class? https://getbootstrap.com/docs/4.0/utilities/text/#text-wrapping-and-overflow

and curious why do we need this?

Reply:
Updated, using the text-wrap now. It was needed because without this the side nav of the wizard looks something likes this, which IMO doesn't look good.
![Screenshot from 2024-07-12 14-09-41](https://github.com/user-attachments/assets/770778fb-1951-4dd4-9cfe-38299e01f6e4)